src/lib/ngc/create-emit-callback.ts

```diff
@@ -1,87 +0,0 @@
-// XX: internal in ngc's `main()`, a tsickle emit callback is passed to the tsc compiler
-// ... blatanlty copy-paste the emit callback here. it's not a public api.
-// ... @link https://github.com/angular/angular/blob/24bf3e2a251634811096b939e61d63297934579e/packages/compiler-cli/src/main.ts#L36-L38
-
-/*
- * @license
- * Copyright Google Inc. All Rights Reserved.
- *
- * Use of this source code is governed by an MIT-style license that can be
- * found in the LICENSE file at https://angular.io/license
- */
-
-// @link https://github.com/angular/angular/blob/24bf3e2a251634811096b939e61d63297934579e/packages/compiler-cli/src/transformers/util.ts#L14
-const GENERATED_FILES = /(.*?)\.(ngfactory|shim\.ngstyle|ngstyle|ngsummary)\.(js|d\.ts|ts)$/;
-
-import * as tsickle from 'tsickle';
-import * as api from '@angular/compiler-cli/src/transformers/api';
-
-// @link https://github.com/angular/angular/blob/83d207d/packages/compiler-cli/src/main.ts#L42-L84
-export function createEmitCallback(options: api.CompilerOptions): api.TsEmitCallback | undefined {
```

Comment:
Is this removing the ability to run the tsickle transforms? If so, why are we removing it?

Reply:
You are right.